### ISSUE TITLE
Change the trip model

### ISSUE THREAD

**Issue by hoanghuynh1995** [npke/trip-hub#3]
I suggest you add departPlace and arrivePlace

**Comment by npke** [comment]
Yes. I had considered it, but sometimes the trip may contain more than 1 arrival place. For example: What if I want to make a trip from Saigon to Dalat and then Nha Trang? Our application won't force the trip to a fixed arrival place. Instead of that, in the trip model we have a field name `places`, it is an array contain all place that the trip plan to. The first element is departure place and the last element is arrival place.
